nautilus_open_any_terminal/open_any_terminal_extension.py

```diff
@@ -24,34 +25,36 @@
 from gi.repository import Gio, GObject, Gtk, Nautilus  # noqa: E402
 
 TERM_WORKDIR_PARAMS = {
-    "alacritty": "--working-directory ",
-    "blackbox": "--working-directory ",
-    "cool-retro-term": "--workdir ",
-    "deepin-terminal": "--work-directory ",
-    "foot": "--working-directory=",
-    "footclient": "--working-directory=",
-    "gnome-terminal": "--working-directory=",
-    "guake": "guake --show --new-tab=",
-    "hyper": "",
-    "kermit": "-w ",
-    "kgx": "--working-directory=",
-    "kitty": "--directory ",
-    "konsole": "--workdir ",
-    "mate-terminal": "--working-directory=",
-    "mlterm": "--working-directory=",
-    "qterminal": "--workdir ",
-    "rio": "--working-dir=",
-    "sakura": "-d ",
-    "st": "-d ",
-    "terminator": "--working-directory=",
-    "terminology": "--current-directory ",
-    "termite": "-d ",
-    "tilix": "-w ",
-    "urxvt": "-cd ",
-    "urxvtc": "-cd ",
-    "wezterm": "start --cwd ",
-    "xfce4-terminal": "--working-directory=",
-    "tabby": "open ",
+    "alacritty": None,
+    "blackbox": ["--working-directory"],
+    "cool-retro-term": ["--workdir", "."],
+    "deepin-terminal": None,
+    "foot": None,
+    "footclient": None,
+    "gnome-terminal": None,
+    "guake": ["--show", "--new-tab=."],
+    "hyper": None,
+    "kermit": None,
+    "kgx": None,
+    "kitty": None,
+    "konsole": None,
+    "mate-terminal": None,
+    "mlterm": None,
+    "qterminal": None,
+    "rio": None,
+    "sakura": None,
+    "st": None,
+    "terminator": None,
+    "terminology": None,
+    "termite": None,
+    "tilix": None,
+    "urxvt": None,
+    "urxvtc": None,
+    "uxterm": None,
+    "wezterm": ["start", "--cwd", "."],
+    "xfce4-terminal": None,
+    "xterm": None,
+    "tabby": ["open"],
```

Comment:
the removed lines contained the parameters. why are they removed?

I am a bit confused about the github-ui, why so much changes are dsplayed.

Reply:
Because they are not needed anymore. The only terminals left that still have these parameters are the ones that don't respect the current directory the parent process chose for them. At least all of them except blackbox can be overridden with a dot path.

Reply:
ok i get the idea. Thats way better... Thanks for explaining.